index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intento de YouTube</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="topNav">
+        <img id="logo" src="./imagenes/logo.jpg" alt="">
+        <div>
+        <input id="search" placeholder="search" type="text">
+        <button>search</button>
+        </div>
+        <div id="profile">
+        <button>notifications</button>
+        <button id="profile">profile</button>
+        </div>
+        
+    </nav>
+    <div id="main">
```

Comment:
try to use Section tag instead of a div

Reply:
Kindly add an explanation because the reason for the required change might not be obvious to others.

Reply:
kindly replace the div with a section tag in order to make your code clear